webapp/angular.json

```diff
@@ -29,14 +29,18 @@
               "src/audio"
             ],
             "styles": [
-              "src/css/inbox.less"
+              "src/css/inbox.less",
+              "node_modules/select2/dist/css/select2.min.css",
```

Comment:
I think we were already including the select2 css directly in inbox.less, was it not enough?